examples/pubsub/pubsub.go

```diff
@@ -0,0 +1,50 @@
+package pubsub
+
+import (
+	"context"
+	"fmt"
+	"github.com/testcontainers/testcontainers-go/wait"
```

Comment:
I'm not sure if this is compliant with the formatting, could you check it?

Reply:
fixed